packages/ui/src/components/molecules/ProductCard/styles.scss

```diff
@@ -168,7 +169,8 @@
   // --------------------------------------------------------
 
   &[data-fs-product-card="out-of-stock"] {
```

Comment:
question: The default `out-of-stock` style should be transparent bkg color, right?
<img width="244" alt="image" src="https://github.com/user-attachments/assets/adee2b37-1e7b-414d-9497-003451a42058">

We only have one `data-fs-product-card` attr called. 🤔 
```suggestion
 [data-fs-product-card="out-of-stock"] {
```


[ProductCard](https://github.com/vtex/faststore/blob/31a958b995b2477f5c15b0b7dd8ec522e47629d9/packages/components/src/molecules/ProductCard/ProductCard.tsx#L39):
 ```
<article
        ref={ref}
        data-fs-product-card={outOfStock ? 'out-of-stock' : ''}
        data-fs-product-card-variant={variant}
        data-fs-product-card-bordered={bordered}
        data-testid={testId}
        {...otherProps}
      >
        {children}
 </article>
```